src/FacebookEventForwarder.js

```diff
@@ -90,9 +98,15 @@
                                 visitorData['external_id'] = selectedIdentity[0].Identity;
                             }
                         }
+                    }
 
-                        fbq('init', settings.pixelId, visitorData);
+                    if (settings.disablePushState === 'True') {

```

Comment:
Nitpick: Stuff like this bothers me since it's not always obvious that `True` will always be that casing. I think this would be a better approach:

```suggestion
                    if (settings.disablePushState.toLowerCase() === 'true') {

```

Reply:
Also, would it make sense to make `True` or `true` a constant? I'm torn since that feels a little excessive.

Reply:
It's definitely bothersome that the server sends it this way to the front end. Since this is how it is across all of our kits, I propose keeping this here, and then creating a ticket to make this change across ALL of our kits.  Thoughts?

Reply:
Makes sense. I'll make the epic.